docs/app/Examples/collections/Message/Variations/MessageInfoPropsExample.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Message } from 'stardust'
+
+const MessageInfoPropsExample = () => (
+  <Message
+    info
+    header='Was this what you wanted?'
+    content="Did you know it's been a while?"
+  />
+)
+
+export default MessageInfoPropsExample
```

Comment:
Without inspecting the code snippet, this example (MessageInfoPropsExample) and the previous (MessageInfoExample) look identical. I think it would be good to add either a title to each example or have the text within the example acknowledge the difference. On first scroll, it just looks a bug in the docs.